index.bs

```diff
@@ -1382,18 +1382,28 @@ Votes</h3>
 	Procedures for <a href="#ACVotes">Advisory Committee votes</a> are described separately.
 
 <h3 id="WGAppeals">
-Appeal of a Chair's Decision</h3>
+Appeal of a Chair's Decision or of a Group Decision</h3>
 
 	Groups resolve issues through dialog.
 	Individuals who disagree strongly with a decision
 	<em class="rfc2119">should</em> register with the Chair any [=Formal Objections=]
 	(e.g., to a decision made as the result of a <a href="#Votes">vote</a>).
 
-	When group participants believe that their concerns are not being duly considered by the group,
+	As detailed in other parts of this document,
+	the [=Chair=] of a [=Working Group=] or [=Interest Group=] has the prerogative
+	to make certain decisions based on their own judgement.
+	Such decisions are called <dfn export>Chair Decisions</dfn>.
+	In contrast,
+	decisions taken by the [=Chair=] of a [=Working Group=] or [=Interest Group=]
+	on the basis of having assessed the [=consensus=] of the group
```

Comment:
A Group Decision might be made as a result of a vote as well as consensus, so add a new line beneath this such as:

```
or following a [=vote=]
```

assuming that decisions arrived at by vote can also be appealed.

Reply:
Discussed at https://www.w3.org/2019/05/22-w3process-minutes.html#item01 and there seems to be consensus that appeals of group decisions arrived at by vote should be permitted.